gtfs-netex-test/gtfsprofile.py

```diff
@@ -469,7 +474,7 @@ def projectStopPlaceToStop(stop_place: StopPlace, transformer: Transformer = Non
                 'zone_id': '',
                 'stop_url': '',
                 'location_type': 1, # Station
-                'parent_station': '',
+                'parent_station': '',  #TODO should be set correctly
```

Comment:
```suggestion
                'parent_station': '',
```
Not a bug, this is correct. GTFS at current writing does not have a parent station of a parent station.